client/landing/jetpack-cloud/components/masterbar/index.jsx

```diff
@@ -18,6 +21,7 @@ import './style.scss';
 
 export default function() {
 	const translate = useTranslate();
+	const user = useSelector( state => getCurrentUser( state ) );
 
 	return (
 		<Masterbar>
```

Comment:
Lets add a className to this master bar. This way we can have any styles not bleed over to the calypso one. 
See https://github.com/Automattic/wp-calypso/pull/39708

Reply:
I've addressed it in ae8ee11